app/profile/shared-games/page.tsx

```diff
@@ -0,0 +1,34 @@
+import { FRIEND_ID_PARAM, STEAM_ID_COOKIE } from "@/constants";
```

Comment:
You can create `/shared-games/{currentUserId}/{comparedUserId}` this file structure.